#935 fix individual training plan

wsbrenk opened the pull request:
1. changes proposed in this pull request:
 
   - fixes issue #935 
   
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)
 
 
2. changelog and release_notes ...

 - [ ] have been updated
 - [ ] do not require update


3. [Optional] suggested person to review this PR @___

akasolace commented:
But I closed this ticket earlier today .... was working fine for what I can tell ...

wsbrenk commented:
this fixes another bug we/i introduced as we changed training and hattrick date handling. the training priority periods which can be set in training prediction table individually for each player do not work without this fix.
